### PR TITLE
Chore: 677 - SonarCloud code smells

### PR DESCRIPTION
**Pull request by bradmartin**
Fixed some code smell warnings per SonarCloud analysis. No functionality should be impacted, just renamed variables in some instances or commented out 'useless' for now until someone can confirm the reason we don't use them in another PR.

### CODE REVIEW

**Comment by finger563** [comment]
@bradmartin are you ready for @p-ranav and me to reveiew this?

File: apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts [hunk 1]
```diff
@@ -517,7 +517,7 @@ export class UpdatesViewModel extends Observable {
     let promises = [];
     const files = [];
     // get the max firmware version for each firmware
-    const maxes = mds.reduce((maxes, md) => {
+    const allMaxes = mds.reduce((maxes, md) => {
```
Comment (finger563) [inline apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts]:
```suggestion
    const reducedMaxes = mds.reduce((maxes, md) => {
```
changing name to be consistent with other renames of similar code

File: apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts [hunk 2]
```diff
@@ -531,7 +531,7 @@ export class UpdatesViewModel extends Observable {
       const fwName = f['_filename'];
       const current = this.currentVersions[fwName];
       const currentVersion = current && current.version;
-      const isMax = v === maxes[fwName];
+      const isMax = v === allMaxes[fwName];
```
Comment (finger563) [inline apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts]:
```suggestion
      const isMax = v === reducedMaxes[fwName];
```
changing name to be consistent with other renames of similar code.

File: apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts (hunk 1 above)
Comment (p-ranav) [reply apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts]: Make sure to change line 534 if committing this suggestion.

**Comment by finger563** [comment]
@p-ranav  looks like our quality gate failed - merge anyway or fix in this branch?

**Comment by p-ranav** [comment]
> @p-ranav looks like our quality gate failed - merge anyway or fix in this branch?

Let's fix it in this branch and then merge.

**Pull request merged**